samples/Exporters/AspNet/Global.asax.cs

```diff
@@ -3,14 +3,12 @@
 using System.Web.Http;
 using System.Web.Mvc;
 using System.Web.Routing;
-using OpenTelemetry.Trace;
 using OpenTelemetry.Trace.Configuration;
 
 namespace OpenTelemetry.Exporter.AspNet
 {
     public class WebApiApplication : HttpApplication
     {
```

Comment:
please try to avoid unrelated changes. While this is good change, its not related to adding net452, so it's best to do separately.

Reply:
ok. I will revert those changes.